tests/test_rabbitmq.py

```diff
@@ -91,6 +91,59 @@ def do_work():
     assert 800 <= success_time - failure_time <= 1200
 
 
+def test_rabbitmq_actors_retry_with_queue_escalation_on_failure(rabbitmq_broker, rabbitmq_worker):
+    for queue_name in ["default.high", "default.medium"]:
+        rabbitmq_broker.declare_queue(queue_name)
+
+    incoming_queues = []
+    escalation_queues_order_arg = {
```

Comment:
escalation_queues_order_arg = {
        "default": "default.medium",
        "default.medium": "default.high",
    }
    something like this should be enougth right ?